lib/instrumentation/decorator.ts

```diff
@@ -0,0 +1,46 @@
+import type { Attributes, SpanKind } from '@opentelemetry/api';
+import { Decorator, decorate } from '../util/decorator';
+import { instrument as instrumentFunc } from '.';
+
+/**
+ * The instrumentation decorator parameters.
+ */
+interface SpanParameters {
```

Comment:
```suggestion
export interface SpanParameters {
```

we probably should have a types.ts file for eyxporting public interfaces